schema/me/artwork_inquiries.js

```diff
@@ -0,0 +1,43 @@
+import gravity from '../../lib/loaders/gravity';
+import Artwork from '../artwork/index';
+import { pageable, getPagingParameters } from 'relay-cursor-paging';
+import {
+  connectionDefinitions,
+  connectionFromArraySlice,
+} from 'graphql-relay';
+import {
+  GraphQLObjectType,
+  GraphQLID,
+  GraphQLNonNull,
+} from 'graphql';
+
+export const ArtworkInquiryType = new GraphQLObjectType({
+  name: 'ArtworkInquiry',
+  fields: () => ({
+    id: {
+      type: GraphQLID,
+    },
+    artwork: {
+      type: new GraphQLNonNull(Artwork.type),
+      resolve: ({ inquireable }) => inquireable,
+    },
+  }),
+});
+
+export default {
+  type: connectionDefinitions({ nodeType: ArtworkInquiryType }).connectionType,
+  args: pageable({}),
+  description: 'A list of the current user’s inquiry requests',
+  resolve: (root, options, request, { rootValue: { accessToken } }) => {
+    if (!accessToken) return null;
+    const { limit: size, offset } = getPagingParameters(options);
+    const gravityArgs = { size, offset, inquireable_type: 'artwork', total_count: true };
```

Comment:
Yeah this is not for now.

However, a total count could definitely change while paging, consider new works being published while paging through works. In fact, it’s entirely realistic that someone would leave a page open for more than a day and then come back to it and page on, in which time many works could have published.

I agree that it’s very unlikely in the case of inquiries, though.